.travis.yml

```diff
@@ -23,17 +23,22 @@ env:
     - secure: "JZRf0nvjA0bNtlHCo2FiF9Xls11cYLoEK4JComWELDq0Qi1wOtlORR9t9RgyOCeldUt0n8ZiDtGVh42jCyI9xqo+lMpzZsp4GWu3apMSiDfFRNhDrakYxU1FMefathMHBCTk8rmq+P1LX7i/V+wPbNyzoSoNzcci3FmEc6eRUEGM/sBgHqH/xyvRFGZnSXbOMTigzgVSruZOVtL6opxHfo1RnxlU/dR5lytKIKm00r+GPK1w9tuwOPTdGpZUaWA6/aoxMQkUNnKqrZq382xBJ6TFgzmkkY80wJFnvchg0NuPqI7oNoA8z30W0vgYbZ7nno3SExjoCYbwCLWPlwIwsCEipEUG1lFPvVwAMELkCWu22JX95e34ZKxd5SR143/509CFMKHK5UrvLqg4tI7EgjrRpLcEI8utvNdjTCp/KUOcXbLq+cw1TMnzDwFuIdgPfsKGrTNreuaiHxCdt2NKuy8oD497WcMOUpq4rWoWXjDXeUfwxOt6GEH1F07Em5Ng0mSiwyCHxiL5XERpYj4lAnSnMBIaIXaHOWlGpron0jAtcpTu3lIY2rF7zdpUFSGNVnaiwwHUD7RDxy96zc/ig0X7DVIofeNTTML/rYsHEbB+hO92sBll+lxEq1xjeAO8Khsp0ohJLOyM3iglBgHxXrSfLhcYk6pmKrOqi1onJK0="
 
 
-jobs:
+matrix:
+#py36 and py37 builds for ppc64le and x64
   include:
+    - env: CONFIG=linux_ppc64le_python3.6 UPLOAD_PACKAGES=False DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
```

Comment:
Do you want UPLOAD_PACKAGES as `False`? We want this package out in the Supplementary Channel, correct?
Maybe change that to `True`?

Reply:
Yes, correct I'll update it.